browserpilot/agents/gpt_selenium_agent.py

```diff
@@ -116,7 +119,6 @@ def __init__(
 
         """Set up the driver."""
         _chrome_options = webdriver.ChromeOptions()
-        _chrome_options.add_argument(f"user-data-dir={user_data_dir}")
```

Comment:
Why take this out? If it's local you might still want to use the `user_data_dir`. How is that handled remotely? 

Maybe the right way to go is to check if the `remote_url` is existent, and if not, then we add the user data dir?

Reply:
Oh, no, I see it later in the diff.